### PR TITLE
Publications search and theme

### PR DESCRIPTION
**Pull request by KMDOFNI**
Re-write Publications search
Re-write News search
Updated site search
Re-write News and Publications facets
Block placements
Edit content type displays
Theme updates

### CODE REVIEW

File: project/config/afbini/config/search_api.index.default_content.yml [hunk 1]
```diff
@@ -124,6 +124,8 @@ field_settings:
 
 
 
+
+
```
Comment (ooneill547) [inline project/config/afbini/config/search_api.index.default_content.yml:128]: Do you know where these big gaps in config files are coming from?

File: project/config/afbini/config/views.view.publication_search.yml [hunk 2]
```diff
@@ -105,8 +104,8 @@ display:
           total_pages: null
           id: 0
           tags:
-            next: 'Next ›'
-            previous: '‹ Previous'
+            next: ››
+            previous: ‹‹
```
Comment (ooneill547) [inline project/config/afbini/config/views.view.publication_search.yml:108]: Was this a requested change? Should it not have the words?

File: project/config/afbini/config/facets.facet.news_publication_date.yml [hunk 3]
```diff
@@ -35,7 +39,7 @@ widget:
     reset_text: 'Show all'
     hide_reset_when_no_selection: false
     soft_limit_settings:
-      show_less_label: 'Show fewer'
+      show_less_label: 'Show less'
```
Comment (ooneill547) [inline project/config/afbini/config/facets.facet.news_publication_date.yml:42]: Should this not be fewer or is it changing in Corp lite?

**Review by ooneill547 — APPROVED** [review]
A few minor things caught my eye that you might want to consider looking at.

File: project/config/afbini/config/search_api.index.default_content.yml (hunk 1 above)
Comment (KMDOFNI) [reply project/config/afbini/config/search_api.index.default_content.yml]: It appears to be a bug on exporting with the solr config, Iv now corrected this file

File: project/config/afbini/config/views.view.publication_search.yml (hunk 2 above)
Comment (KMDOFNI) [reply project/config/afbini/config/views.view.publication_search.yml:108]: I think this is correct, next and previous should be blank and First and Last are to have the words

File: project/config/afbini/config/facets.facet.news_publication_date.yml (hunk 3 above)
Comment (KMDOFNI) [reply project/config/afbini/config/facets.facet.news_publication_date.yml:42]: It could be changing , Nisra has it set to less

**Pull request merged**